README.md

```diff
@@ -18,21 +18,21 @@ Inclue or extend our colour configuration and add our components to Tailwind's c
 
 ```js
 // tailwind.config.js
-import colors from '@nearform/quantum/dist/colors'
 
 module.exports = {
-    content: [
-        "./node_modules/@nearform/quantum/**/*.js"
-    ],
-    theme: {
-        colors,
-    }
-};
+  content: [
+    //your content files...
+    './node_modules/@nearform/quantum/**/*.js'
+  ],
+  // The rest of your config... ,
```

Comment:
I suggest removing this comment as what follows is actually key to using this lib as a plugin.